uber/site_sections/groups.py

```diff
@@ -103,6 +103,7 @@ def unapprove(self, session, id, action, email, convert=None, message=''):
                     message = 'Group declined and emails sent to attendees'
                     attendee.paid = c.NOT_PAID
                     attendee.badge_status = c.NEW_STATUS
+                    attendee..badge_cost = c.INITIAL_ATTENDEE
```

Comment:
This looks like the right place to make the change, but I think we want to use the `overridden_price` attribute:
```python
attendee.overridden_price = c.INITIAL_ATTENDEE
```

I think @kitsuta wrote most of the badge pricing code, so she might have a better idea.

Reply:
That does seem better...changing now